doc/kubernetes/modules/ROOT/partials/executor-jgroups-thread-calculation.adoc

```diff
@@ -0,0 +1,2 @@
+Due to the bug https://issues.redhat.com/browse/ISPN-14780[ISPN-14780] there is currently no known way to configure the size different from the default JGroups 100 max threads, so assuming a Keycloak cluster with 4 Pods, each Pod shouldn't have more than 25 worker threads.
```

Comment:
That is not entirely true since you can still configure the thread pool in the extended stack. As an example
```xml
    <jgroups>
        <stack name="kubernetes-with-fdsock" extends="kubernetes">
            <TCP thread_pool.max_threads="${env.JGROUPS_MAX_THREADS:200}" />
            <!-- When using an embedded stack, replacement is done by Infinispan which requires environment variables to be
            prefixed with ".env", while JGroups falls back to environment variables without the prefix.
            The Keycloak Operator passes this information in an enviornment variable `jgroups.dns.query`.
            See https://github.com/keycloak/keycloak/issues/21830 for a discussion.
            -->
            <dns.DNS_PING dns_query="${env.jgroups.dns.query}" />
            <!-- Workaround for problems with FD_SOCK2, which might fail when the other nodes are not ready yet.
            See https://github.com/keycloak/keycloak/issues/21092 -->
            <FD_SOCK stack.combine="REPLACE" stack.position="FD_SOCK2"/>
        </stack>
    </jgroups>
```

Reply:
@pruivo - I tried this. I debugged that the value was correctly picked up in `StringPropertyReplacer`, but it never made it to the `TCP` instance, where the thread pool was still set up to be 100 threads only. 

So I guess I'll wait until ISPN-14780 fixed in an upcoming release.

![image](https://github.com/keycloak/keycloak-benchmark/assets/3957921/1f967408-30c5-45f5-979f-227be9633a93)